fedimint-dbtool/README.md

```diff
@@ -0,0 +1,42 @@
+# `dbtool`
+
+**CAUTION: `dbtool` is a potentially destructive expert-tool, handle with care and if in doubt make a copy of your
+database.**
+
+For database layout see [`database.md`](../docs/database.md), but verify the information in the code before taking
+potentially destructive action.
+
+## Usage
+```
+$ dbtool --help
+Tool to inspect and manipulate rocksdb databases. All binary arguments (keys, values) have to be hex encoded
```

Comment:
Do you have a command for taking some data you want to write and hex encoding it?

Reply:
```bash
$ echo -n "my binary data ..." | xxd -ps -c 0
6d792062696e6172792064617461202e2e2e
```

But tbh, whenever I was writing I had to manually construct these hex strings anyway by encoding the right integers etc.

Reply:
added a section on it